### PR TITLE
Various fixes.

### LINKED ISSUE #2048: [Improvement]: Consistency between Vehicle spawn tabs
### Which feature

The New and Personal Vehicle tabs have an option to preview the vehicle and determine if you want to be teleported into the vehicle, as well as New having Spawn Max and Personal having Spawn Cloned, but those are specific to those features. With the recent introduction of #2042, I feel each vehicle box should conform to the same standard. No more than 30 items per box, and we at least show Preview and Spawn In checkboxes for both the XML and the Persist tab. The Personal vehicle tab also shows vehicles that should not be spawned as personal vehicles, such as the Avenger, acid lab bike, Terrorbyte, and others like the RC Bandito and the Invade and Persuade tank. We should only show them if Spawn Clone is checked, as we could potentially get the user flagged by Rockstar if they assign their personal vehicle slot to a vehicle that can't normally be a personal vehicle. 

### Reason

Consistency.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gir489returns**
Closes #2048 missed issue.
Reverted #2010's Anti-AFK code.
Fixed #2040 not using the tunables service when applicable.

### CODE REVIEW

File: src/views/vehicle/spawn/view_pv.cpp [hunk 1]
```diff
@@ -8,6 +8,32 @@
...
 namespace big
 {
+	bool is_blacklist_vehicle(Hash vehicle_hash)
+	{
+		switch (vehicle_hash)
+		{
+			case RAGE_JOAAT("avenger"):
+			case RAGE_JOAAT("avenger3"):
+		    case RAGE_JOAAT("hauler2"):
+		    case RAGE_JOAAT("phantom3"):
```
Comment (Yimura) [inline src/views/vehicle/spawn/view_pv.cpp]: Mixed indentation

File: src/backend/looped/self/toggle_passive.cpp [hunk 2]
```diff
@@ -21,15 +22,15 @@ namespace big
...
+			*scr_globals::passive.as<PBOOL>()                 = TRUE;
 		}
 
 		virtual void on_disable() override
 		{
-			*script_global(1574582).as<bool*>() = false;
+			*scr_globals::passive.as<PBOOL>() = FALSE;
 			NETWORK::SET_LOCAL_PLAYER_AS_GHOST(false, false);
-			*scr_globals::tuneables.at(27371).as<int*>() = 30000;
+			*g_tunables_service->get_tunable<int*>(-29732167) = 30000;
```
Comment (Yimura) [inline src/backend/looped/self/toggle_passive.cpp:33]: What is this `-29732167`? Any way on finding it again if it changes?

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/backend/looped/self/toggle_passive.cpp (hunk 2 above)
Comment (gir489returns) [reply src/backend/looped/self/toggle_passive.cpp:33]: It’s just the JOAAT’s integer representation. It’s what was in the tunables file. I tried looking for JOAATs that matched it, but all I could find were collisions. The tunable service will auto update it with the matching JOAAT value.

File: src/views/vehicle/spawn/view_pv.cpp (hunk 1 above)
Comment (gir489returns) [reply src/views/vehicle/spawn/view_pv.cpp:18]: Nit picking.

**Review by Yimura — APPROVED** [review]

**Pull request merged**